src/seedu/addressbook/AddressBook.java

```diff
@@ -210,13 +210,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        loopProgram();
+    }
+
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.